Sources/VitaminUIKit/Components/Snackbar/README.md

```diff
@@ -191,13 +191,14 @@ To ease the use of `VitaminSnackbar`, Vitamin provides a `UIViewController` exte
 
 The first parameter is a `VitaminSnackbar`.
 
-There are two others parameters:
-- `maxWidth`: it is optioanl, and default value is `nil`. If you do not set it, the max width of the snackbar will be the width of the View of the controller, with a margin of 10 on each side.
-- bottomMargin: snackbar will always be centered horizontally in view, but you can decide the space between the bottom of the view and the bottom of the snackbar. Default value is 10
+There is just one extra parameter:
+- bottomMargin: snackbar will always be centered horizontally in view, but you can set the space between the bottom of the view and the bottom of the snackbar. Default value is 10
 
 This method has also a great advantage: it automatically handles the fact that only one snackbar should be displayed.
 If you call it while a snackbar is already displayed, it will first dismiss it, then display the new one.
 
+If your snackbar has a `maxWidth`, it wil be applied. But if it is not set, a default max width will be applied, which is the width of the controller's view minus a padding of 10px on each side.
```

Comment:
```suggestion
If your snackbar has a `maxWidth`, it will be applied. But if it is not set, a default max width will be applied, which is the width of the controller's view minus a padding of 10px on each side.
```